fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRun.java

```diff
@@ -122,6 +124,31 @@ public ConnectContext getRunCtx() {
     }
 
     public TaskRunStatus getStatus() {
+        if (status == null) {
+            return null;
+        }
+        switch (status.getState()) {
+            case RUNNING:
+                if (runCtx != null) {
+                    StmtExecutor executor = runCtx.getExecutor();
+                    if (executor != null) {
+                        Long jobId = executor.getCoordinator().getJobId();
```

Comment:
Is there any change executor.getCoordinator() returns null?

Reply:
Defensive programming, because the execution of task run changes the state to running firstly, and then creates the stmtexecutor. If the state  changes to running, but the stmtexecuor is not created, querying the taskrun state at this time may result in null

Reply:
yes, i will check this